### PR TITLE
Changelog for 1.6.9, 1.6.10, and 1.7.0

### PR DESCRIPTION
**Pull request by 5chdn**
~~WIP~~

### CODE REVIEW

**Comment by 5chdn** [comment]
let me know if any killer features are missing from the summary and I can draft a short text.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,439 @@
+## Parity [v1.7.0](https://github.com/paritytech/parity/releases/tag/v1.7.0) (2017-07-17)
+
+[WIP] @TODO Random list of new features
+
+- Light client support
```
Comment (rphmeier) [inline CHANGELOG.md]: would like to clarify experimental here. it will likely be a few releases before it feels mature

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,439 @@
...
+- Public node support
+- Run contracts compiled in WASM
```
Comment (arkpar) [inline CHANGELOG.md]: Requires some documentation at least. @NikVolf?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,439 @@
...
+- DApps and RPC server merge
```
Comment (arkpar) [inline CHANGELOG.md]: Does it affect users. What are the changes to the CLI/settings?

File: CHANGELOG.md (hunk 2 above)
Comment (NikVolf) [inline CHANGELOG.md]: Yes, will do
also requires experimental notice

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
-- DApps and RPC server merge
-- Cancel propagated transactions
-- Export accounts from the wallet
-- PoA/Kovan validator contract warp sync
+- **Experimental [Light client](https://github.com/paritytech/parity/wiki/The-Parity-Light-Protocol-(PIP)) support**. Start Parity with `--light` to enable light mode.
+- **Experimental web wallet**. A hosted version of Parity that keeps the keys and signs transactions using your browser storage. Try it at https://wallet.parity.io or run your own with `--public-node`.
+- **WASM contract support**. Private networks can run contracts compiled into WASM bytecode. _More information and documentation to follow_.
+- **DApps and RPC server merge**. DApp and RPC are now available through a single API endpoint. DApp server related settings are deprecated.
+- **Export accounts from the wallet**. Backing up your keys can now simply be managed through the wallet interface.
+- **PoA/Kovan validator set contract**. PoA networks now support validator set management via a smart contract.
```
Comment (rphmeier) [inline CHANGELOG.md]: technically this feature existed, the key point now is that it is mostly stable and supports warp and potentially light sync

**Pull request merged**